### ISSUE TITLE
Question about the setup instructions

### ISSUE THREAD

**Issue by bartbutenaers** [mikejac/node-red-contrib-google-smarthome#464]
Hi guys,

I have a question about the [setup](https://github.com/mikejac/node-red-contrib-google-smarthome/blob/master//docs/setup_instructions.md) instructions, especially this part:
> You need to be able to forward incoming traffic from the internet to a specific port on your host. This may be difficult if your ISP uses carrier-grade NAT, if you are on a shared network (e.g. in dormitories) or if you can'tconfigure port forwarding on your router. In this case you can use services like [ZeroTier](https://www.zerotier.com/) or [Cloudflare Tunnel](https://www.cloudflare.com/products/tunnel/).

We had last week a [discussion](https://discourse.nodered.org/t/ideas-for-a-simple-setup-of-a-free-tier/84004/26?u=bartbutenaers) on Discourse about using ZeroTier.  But I am stuck with port 3001, as I explained on the ZeroTier [forum](https://discuss.zerotier.com/t/send-http-request-from-outside-the-virtual-network/17594).

I am now wondering if you find a way to stop port forwarding of port 3001, and use ZeroTier instead?

Thanks!!!
Bart

**Comment by Caprico85** [comment]
Hi Bart!

Sorry, that was my mistake. Unfortunately you cannot use Zerotier.

There have been reports [HERE](https://github.com/mikejac/node-red-contrib-google-smarthome/issues/446) that Cloudflare Tunnel can be used. When I wrote the docs, I wanted to include at least one alternative to CloudFlare and after some research, I found ZeroTier.

I admit that I added it to the docs without testing it. Now, after some more research now I realized that I was wrong. It seems that ZeroTier can only be used to connect between two private networks. But ZeroTier doesn't provide a public, internet-facing endpoint to a service like Cloudflare does.

So, in the end, you still need to open up a port. Either on your own machine or via tunneling on somebody else's machine.

I have now removed ZeroTier from the docs.
